myapp/spec/system/tasks_spec.rb

```diff
@@ -3,8 +3,26 @@
 require "rails_helper"
 
 RSpec.describe "Tasks", type: :system do
-  before do
-    @task = Task.create!(title: "test task title", description: "test task description")
+  
+  # Create dummy data
+  let!(:tasks) do
+    (1..9).map do |i|
+      Task.create(title: "test title #{i}", description: "test description #{i}", created_at: i.days.ago)
+    end
+  end
+
+  let!(:task) { tasks.first }
+
+  describe "check order by creation date" do
+    before do
+      visit tasks_path
+    end
+
+    it "expect return desceding order" do
+      for i in 1..9
```

Comment:
using loop inside it block is not recommended
Since you already have tasks variables, I think it's better to use tasks with each_with_index